### ISSUE TITLE
Codenvy Installation manager backup fails on long file path in workspace

### ISSUE THREAD

**Issue by dmytro-ndp** [codenvy/codenvy#689]
IM CLI error log:

> [root@ip-172-31-25-238 backups]# codenvy backup
> 
> [CODENVY] {
> 
> "message" : "Command execution fail. Error: Can't execute command 'if sudo test -f /home/centos/codenvy/backups/codenvy_backup_03-Sep-2016_14-30-12.tar; then sudo tar -C /home/codenvy/codenvy-data -rf /home/centos/codenvy/backups/codenvy_backup_03-Sep-2016_14-30-12.tar fs/.;else sudo tar -C /home/codenvy/codenvy-data -cf /home/centos/codenvy/backups/codenvy_backup_03-Sep-2016_14-30-12.tar fs/.;fi;'. Output: ; Error: tar: fs/./7c/7f/25/workspaceu3qht8l1ardzutr5/ELink1/src/main/webapp/ext/plugins/bower_components/sing-wrapbootstrap-3.1.1/angular/backgrid-paginator/api/extjs/resources/themes/images/default/panel-header/panel-header-default-framed-collapsed-bottom-corners.gif: file name is too long (cannot be split); not dumped\ntar: fs/./78/ba/f6/workspacem5xtvcowdejxxci9/Elink/src/main/webapp/ext/plugins/bower_components/sing-wrapbootstrap-3.1.1/angular/backgrid-paginator/api/extjs/resources/themes/images/default/panel-header/panel-header-default-framed-collapsed-bottom-corners.gif: file name is too long (cannot be split); not dumped\ntar: fs/./8c/29/7b/workspacel626w8c0950u4aj7/elnk-poc/src/main/webapp/ext/plugins/bower_components/sing-wrapbootstrap-3.1.1/angular/backgrid-paginator/api/extjs/resources/themes/images/default/panel-header/panel-header-default-framed-collapsed-bottom-corners.gif: file name is too long (max 256); not dumped\ntar: Exiting with failure status due to previous errors.",
> 
> "status" : "ERROR"

In version starting from 5.0.0-M2-SNAPSHOT, this issue depends on #465.

**Comment by dmytro-ndp** [comment]
Can't reproduce this bug in GNU tar version 1.26.

**Comment by bmicklea** [comment]
@dmytro-ndp why was this merged into M7 and not M6?

**Comment by dmytro-ndp** [comment]
@bmicklea: milestone was fixed.
